.gitattributes

```diff
@@ -0,0 +1,4 @@
+# get php syntax highlighting for these file extensions
+*.module linguist-language=php
+*.install linguist-language=php
+*.test linguist-language=php
```

Comment:
```suggestion
*.test linguist-language=php
*.profile linguist-language=php
```

Reply:
Yep, nice catch, let's also add .profile to the list. :+1: